airflow/contrib/operators/gcp_transfer_operator.py

```diff
@@ -0,0 +1,856 @@
+            gcp_conn_id=self.gcp_conn_id,
+        )
+        return hook.update_transfer_job(
+            job_name=self.job_name,
+            body=self.body,
+        )
+
+
+class GcpTransferServiceJobDeleteOperator(BaseOperator):
+    """
+    Delete a transfer job.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GcpTransferServiceJobDeleteOperator`
+
+    :param job_name: The name of the operation resource to be cancelled.
+    :type job_name: str Name of the transfer job. Required.
+    :type gcp_conn_id: str
+    :param project_id: Optional, Google Cloud Platform Project ID.
+    :type project_id: str
+    :param gcp_conn_id: The connection ID used to connect to Google Cloud
+        Platform.
+    :type gcp_conn_id: str
+    :param api_version: API version used (e.g. v1).
+    :type api_version: str
+    """
+    # [START gcp_transfer_job_delete_template_fields]
+    template_fields = ('job_name', 'project_id', 'gcp_conn_id', )
+    # [END gcp_transfer_job_delete_template_fields]
+
+    @apply_defaults
+    def __init__(self,
+                 job_name,
+                 gcp_conn_id='google_cloud_default',
+                 api_version='v1',
+                 project_id=None,
+                 *args,
+                 **kwargs):
+        super(GcpTransferServiceJobDeleteOperator, self)\
+            .__init__(*args, **kwargs)
+        self.job_name = job_name
+        self.project_id = project_id
+        self.gcp_conn_id = gcp_conn_id
+        self.api_version = api_version
+        self._validate_inputs()
+
+    def _validate_inputs(self):
+        if not self.job_name:
+            raise AirflowException("The required parameter 'job_name' "
+                                   "is empty or None")
+
+    def execute(self, context):
+        self._validate_inputs()
+        hook = GCPTransferServiceHook(
+            api_version=self.api_version,
+            gcp_conn_id=self.gcp_conn_id,
+        )
+        hook.delete_transfer_job(
+            job_name=self.job_name,
+            project_id=self.project_id
+        )
+
+
+class GcpTransferServiceOperationGetOperator(BaseOperator):
+    """
+    Gets the latest state of a long-running operation in Google Storage Transfer
+    Service.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GcpTransferServiceOperationGetOperator`
+
+    :param operation_name: Name of the transfer operation. Required.
+    :type operation_name: str
+    :param gcp_conn_id: The connection ID used to connect to Google
+        Cloud Platform.
+    :type gcp_conn_id: str
+    :param api_version: API version used (e.g. v1).
+    :type api_version: str
+    :return: Google Storage Transfer service object
+    :rtype: dict
+    """
+    # [START gcp_transfer_operation_get_template_fields]
+    template_fields = ('operation_name', 'gcp_conn_id', )
```

Comment:
```suggestion
    template_fields = ('operation_name', 'gcp_conn_id')
```

Applicable to all operators, unless there's a reason behind it that I'm unaware of.

Reply:
I apply your suggestion.